.github/workflows/run-eval-pf-pipeline.yml

```diff
@@ -62,7 +68,7 @@ jobs:
         pf connection create --file ../connections/azure_openai.yaml --name open_ai_connection --set api_key=${{ env.OPENAI_KEY }} api_base=${{ env.OPENAI_ENDPOINT }}
         pf connection create --file ../connections/azure_ai_search.yaml --name ai_search_connection --set api_key=${{ env.AISEARCH_KEY }} api_base=${{ env.AISEARCH_ENDPOINT }}
     - name: Test promptflow run
-      run: pf flow test --flow .
+      run: pf flow test --flow . --inputs search_index="${{ env.AISEARCH_INDEX }}"
     - name: Run promptflow batch
       run: pf run create --file batch_run.yaml --name batch_run
```

Comment:
I think we should also use the input in the batch run:

`--inputs search_index="${{ env.AISEARCH_INDEX }}"`

Reply:
apparently it doesn't, or I'm doing it wrong. Let's have a look tomorrow.

Reply:
as discussed, removed variable.